pyfar/signals/files.py

```diff
@@ -0,0 +1,516 @@
+"""
+This module contains anechoic audio content and impulse responses for listening
+and illustration. Note that each file has a separate license that is listed
+below.
+"""
```

Comment:
What about a short example on quick listening?
```suggestion

Quick listening is, e.g., possible with `sounddevice
<https://python-sounddevice.readthedocs.io/en/0.4.4/index.html>`_ installed:

>>> import pyfar as pf
>>> import sounddevice as sd
>>> # Load, illustrate and play speech signal
>>> speech = pf.signals.files.speech()
>>> pf.plot.spectrogram(speech)
>>> sd.play(speech.time.T, speech.sampling_rate)
"""
```

Reply:
absolutely - added that